File: zuritym/zuritym-backend/app/Models/PaymentMethod.php
<?php
namespace App\Models;
use Illuminate\Database\Eloquent\Model;
class PaymentMethod extends Model {
    protected $fillable = ['name','slug','icon','min_withdrawal','max_withdrawal','conversion_rate','fields','is_active','sort_order'];
    protected $casts = ['is_active'=>'boolean','fields'=>'array'];
}

File: zuritym/zuritym-android/app/src/main/java/com/zuritym/app/models/PaymentMethod.java
package com.zuritym.app.models;
import com.google.gson.annotations.SerializedName;
public class PaymentMethod {
    @SerializedName("id")             public int id;
    @SerializedName("name")           public String name;
    @SerializedName("slug")           public String slug;
    @SerializedName("icon_url")       public String iconUrl;
    @SerializedName("min_withdrawal") public double minWithdrawal;
    @SerializedName("conversion_rate") public double conversionRate;
}